code/modules/surgery/organs/tongue.dm

```diff
@@ -15,6 +15,7 @@
 		/datum/language/common,
 		/datum/language/draconic,
 		/datum/language/monkey,
+		/datum/language/narsie,
```

Comment:
this whole list should really be made a define or something in the actual `language` folder but that's kind of unrelated to this PR